cves/CVE-2009-3095.yml

```diff
@@ -32,7 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+        This vulnerability allowed authorized attacks to send commands to an
+        FTP server due to inadequate input validation.
```

Comment:
Describe how the vulnerability allowed authorized attacks to send commands

Reply:
Updated commit better describes how attacks are created